### PR TITLE
Added information for the MeteoAlarm Sensor

### PR DESCRIPTION
**Pull request by rolfberkenbosch**
**Description:**
Documentation about a new sensor meteoalarm to get weather alerts in Europe.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23663

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/meteoalarm.markdown [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+date: 2019-05-03 16:50
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: meteoalert.png
+ha_category: Sensor
+ha_release: 0.93
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/meteoalarm.markdown]: Please remove this

File: source/_components/meteoalarm.markdown [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+sensor:
+  - platform: meteoalarm
+    country: 'NL' (use your own country digits)
+    province: 'Groningen' (use your own province name)
+    language: 'ne-NL' (use your language)
```
Comment (klaasnicolaas) [inline source/_components/meteoalarm.markdown]: Only required variables

File: source/_components/meteoalarm.markdown [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+The sensor state shows if applicable the warning message. The details are available as attribute.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+sensor:
+  - platform: meteoalarm
+    country: 'NL' (use your own country digits)
```
Comment (klaasnicolaas) [inline source/_components/meteoalarm.markdown]:
```suggestion
    country: 'NL'
```

File: source/_components/meteoalarm.markdown [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    province: 'Groningen' (use your own province name)
```
Comment (klaasnicolaas) [inline source/_components/meteoalarm.markdown]:
```suggestion
    province: 'Groningen'
```

File: source/_components/meteoalarm.markdown (hunk 2 above)
Comment (klaasnicolaas) [inline source/_components/meteoalarm.markdown]:
```suggestion
    language: 'ne-NL'
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/meteoalarm.markdown [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    language: 'ne-NL' (use your language)
+```
+
+{% configuration %}
+name:
+  description: Sensor name
+  required: false
+  default: "meteoalarm"
```
Comment (klaasnicolaas) [inline source/_components/meteoalarm.markdown]:
```suggestion
  default: meteoalarm
```

File: source/_components/meteoalarm.markdown [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+country:
+  description: The 2 digits of your country
+  required: true
+  type: string
+propvince:
+  description: The province
+  required: true
+  type: string
+language:
+  description: The 4 digits our your language, please be aware that this is only possible in the current country. So 'ne' is only possible in Netherlands. Possible options is: bu, bs, ce, da, de, ee, en, es, ga, ca, su, fr, gr, he, hr, ma, is, it, li, la, sr, mk, ma, ne, no, po, ro, cp, sv, sl, eu.
```
Comment (klaasnicolaas) [inline source/_components/meteoalarm.markdown]:
```suggestion
  description: "The 4 digits our your language, please be aware that this is only possible in the current country. So 'ne' is only possible in Netherlands. Possible options is: bu, bs, ce, da, de, ee, en, es, ga, ca, su, fr, gr, he, hr, ma, is, it, li, la, sr, mk, ma, ne, no, po, ro, cp, sv, sl, eu."
```

Perhaps better to place all possible options in a list elsewhere on the page

File: source/_components/meteoalarm.markdown [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+  required: false
+  type: string
+  default: 'en'.
```
Comment (klaasnicolaas) [inline source/_components/meteoalarm.markdown]:
```suggestion
  default: 'en'
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by rolfberkenbosch** [comment]
I'm sorry guys that i have added so much pull requests. I made a few mistakes, the last one should no be good.

**Comment by fabaff** [comment]
> I'm sorry guys that i have added so much pull requests.

Don't worry, we will squash the commits when we are merging.

**Comment by rolfberkenbosch** [comment]
@klaasnicolaas can you approve my last commit, because the binary sensor has been approved.

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**